app/models/User.php

```diff
@@ -63,6 +63,11 @@ public function isAdmin()
     public function talks()
     {
         return $this->hasMany('Talk', 'author_id');
+        /* @todo can we do this somehow?
+            ->orderBy(function ($talk) {
+                return strtolower($talk->current()->title);
+            });
+        */
```

Comment:
I don't think you can do without adding a raw query clause or loading everything into memory first unfortunately.

You could add this:

```
public function getTalksAttribute()
{
    return $this->talks()->get()->sortBy(function ($talk) {
        return strtolower($talk->current()->title);
    });
}
```

...which would override the standard behavior of calling just `talks` and having it pass through directly to the relationship method. I do that pretty often.